src/App.tsx

```diff
@@ -1,39 +1,36 @@
 import React from 'react';
+import AddButton from './components/layout/AddButton';
 import { Button, Grid, Icon } from '@mui/material';
 import Navbar from './components/layout/Navbar';
 import StudentSummary from './components/layout/StudentSummary';
 import StudentListContainer from './containers/StudentTableContainer';
-// import Forms from './components/Forms';
+import StudentForm from './components/studentForm/StudentForm';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Switch, useNavigate, Router, Route, Routes } from 'react-router-dom';
+
 import './App.css';
 
 function App() {
+  let navigate = useNavigate();
+
+  const handleClick = async () => {
+    console.log('here');
+    navigate('/StudentForm', { state: 'abcd' });
```

Comment:
Student form is not a correct now. 
it should be addStudent or add-student or student